crates/ra_assists/src/handlers/fill_match_arms.rs

```diff
@@ -56,31 +50,115 @@ pub(crate) fn fill_match_arms(ctx: AssistCtx) -> Option<Assist> {
     }
 
     let db = ctx.db;
-
```

Comment:
Before we call `ctx.add_assist`, we should check if all arms are already present, otherwise, a do-nothing assist is suggested:

![image](https://user-images.githubusercontent.com/1711539/76850807-69c9fa80-6848-11ea-85f5-64a7cadfd44a.png)

Reply:
Even if everything is included but the `_`?

Reply:
Not sure I understand the question, but if all cases are covered, one does not need `_`.